models/data.js

```diff
@@ -253,34 +238,38 @@ reader_norm(exports.math, "models/math/norm/", function (element) {
   return normalize_common(element);
 });
 
-var outs = {
+let outs = {
   d: exports.date,
   y: exports.year,
   m: exports.math,
-  t: exports.trivia
+  t: exports.trivia,
 };
-var callbacks = {
+let callbacks = {
   d: normalize_common,
   y: normalize_common,
   m: normalize_common,
-  t: normalize_common
+  t: normalize_common,
 };
 reader_manual(outs, "models/manual/", callbacks);
 
 // check for missing entries
 (function () {
-  var configs = [
+  let configs = [
     { category: "math", data: exports.math, min: 0, max: 251 },
     { category: "trivia", data: exports.trivia, min: 0, max: 251 },
     { category: "date", data: exports.date, min: 1, max: 367 },
-    { category: "year", data: exports.year, min: -100, max: 2050 }
+    { category: "year", data: exports.year, min: -100, max: 2050 },
   ];
 
   _.each(configs, function (config) {
     _.each(_.range(config.min, config.max), function (index) {
       if (!config.data[index] || config.data[index].length === 0) {
-        console.log("Missing: " + config.category + ": " + index);
+        console.warn(`Missing: ${config.category} : ${index}`);
       }
     });
   });
 })();
+
+exports.reader_norm = reader_norm;
+exports.reader_manual = reader_manual;
+exports.normalize_common = normalize_common;
```

Comment:
can you change this to:

```suggestion
module.exports = {
  reader_norm,
  reader_manual,
  normalize_common
}
```

if there's anything else exported in the file, let's move the exporting to the bottom as well.

Reply:
Okay, I'll keep that in mind for future reference. Thanks for reviewing this @mmmaaatttttt! I've made the above changes and pushed it to my version of the branch. 

Can we merge this PR? I'll try look for the differences with PR #34 and #35.

Reply:
Hey @stephenisau, I don't see any new commits on this branch. Did you push up to the correct branch? The PR will auto-update with any new commits when you push them up. Once the commit is on this branch i'll merge :)

Reply:
@mmmaaatttttt just saw this but I think my commit was stuck on my local branch. I just pushed this up to the git branch on my repo!